contrib/gitian-debian/build.sh

```diff
@@ -36,6 +55,7 @@ Architecture: amd64
 Description: Bitcoin XT is a fully verifying Bitcoin node implementation, based on the sources of Bitcoin Core.
 Maintainer: Mike Hearn <hearn@vinumeris.com>
```

Comment:
Sure, no problem.  I'll update the TODO list also to remove what's already completed.